src/apps/chifra/cmd/serve/exec/admin_pins.go

```diff
@@ -0,0 +1,62 @@
+package serveRoutes
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/cmd/pins"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/pinlib"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/pinlib/manifest"
+)
+
+var ErrMultipleModes = errors.New("only one of ?list or ?init is supported")
```

Comment:
Aren't these duplicated in `pins_run.go`? It's okay if they are for now, but eventually, I'll want to consolidate these and bring them under the auto-code-generation tool. Perhaps you're already anticipating that?

Reply:
Yes, they're almost the same. I could have used the ones declared in `pins_run`, but they mention `--option`, which can be misleading when using API (it would be `?option`). So either we remove `--` from the errors or leave it and hope we won't confuse API consumers that much.

Reply:
I would argue to leave it for now. Once things settle down, error messages become something generated from the auto-generator. At that time, we can decide and easily change all `--` to `?`. Until then, just leave them all with `--`.